packages/framework-provider-local-infrastructure/src/test-helper/local-queries.ts

```diff
@@ -0,0 +1,13 @@
+export class LocalQueries {
+  constructor() {}
+
+  public async events(primaryKey: string, latestFirst = true): Promise<Array<unknown>> {
+    // TODO implement method
+    return []
```

Comment:
primaryKey param should be transformed into provider-agnostic params. This method should be implemented in a future PR